Essentials/src/messages.properties

```diff
@@ -46,8 +46,8 @@ broadcast=\u00a7r\u00a76[\u00a74Broadcast\u00a76]\u00a7a {0}
 buildAlert=\u00a74You are not permitted to build.
 bukkitFormatChanged=Bukkit version format changed. Version not checked.
 burnMsg=\u00a76You set\u00a7c {0} \u00a76on fire for\u00a7c {1} seconds\u00a76.
-cannotStackMob=\u00a74You do not have permission to stack multiple mobs.
```

Comment:
Your script seems to have different opinions on how to alphabetically sort the keys - if you could minimise the changes to only include the new keys then that would be ideal (perhaps a revert and manually add new keys in the right places).